calico/felix/test/test_fiptgenerator.py

```diff
@@ -365,63 +361,68 @@
     '--append felix-from-abcd --jump MARK --set-mark 0/0x1000000',
     # From chain polices the MAC address.
     '--append felix-from-abcd --match mac ! --mac-source aa:22:33:44:55:66 '
-               '--jump DROP -m comment --comment '
-               '"Incorrect source MAC"',
+    '--jump DROP -m comment --comment '
```

Comment:
pep8 produces warning if trying to indent second line. it can be resolved by using common indentation for wrapping long lines (visually separated by empty lines) or by using parenthesis. Not sure which style is preferable.